projects/@crucible-common/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@cmusei/crucible-common",
-  "version": "0.0.8",
+  "version": "0.0.9",
```

Comment:
this is not necessary - releasing the plugin updates this.  but it's fine, it'll just overwrite the same thing

Reply:
whoops! thanks for the info.